### ISSUE TITLE
Execution when more confirmations than required

### ISSUE THREAD

**Issue by harzo** [gnosis/MultiSigWallet#103]
Is there a reason why this condition for checking Confirmation count is equal but not greater than `required` number? 
https://github.com/gnosis/MultiSigWallet/blob/886eb7761ee18484fa4e597f14c6e971bf9e7a66/contracts/MultiSigWallet.sol#L276

In my case I was unable to repeat Execution of transaction which originally failed when I got more confirmations than required. Only after revocation of redundant confirmations Execution was available again.

Is it protection against Execution of failing transaction for each additional Confirmation?

**Comment by Georgi87** [comment]
I agree this would be a useful improvement. The new designed multisig contract has this included: https://github.com/gnosis/safe-contracts/blob/master/contracts/GnosisSafeTeamEdition.sol#L85

I keep this issue open.
